azure-pipelines.yml

```diff
@@ -303,7 +303,7 @@ steps:
 
 
 - bash: |
-    cd $BUILD_DIR && cmake  -G "Visual Studio 15 2017 Win64" -DBoost_DEBUG=1 -DBUILD_TESTING=ON -DWITH_tests=ON -DWITH_examples=ON -DCMAKE_CXX_FLAGS_RELEASE=-DCGAL_NDEBUG -DCMAKE_PREFIX_PATH="C:\Miniconda\Library" $SOURCE_DIR  && ctest --build-config "Release" -L $(PACKAGE)
+    cd $BUILD_DIR && cmake  -G "Visual Studio 15 2017 Win64" -DBUILD_TESTING=ON -DWITH_tests=ON -DWITH_examples=ON -DCMAKE_CXX_FLAGS_RELEASE=-DCGAL_NDEBUG -DCMAKE_PREFIX_PATH="C:\Miniconda\Library" $SOURCE_DIR  && ctest --build-config "Release" -L $(PACKAGE) -VV
```

Comment:
```suggestion
    cd $BUILD_DIR && cmake  -G "Visual Studio 15 2017 Win64" -DCGAL_HEADER_ONLY=ON -DBUILD_TESTING=ON -DWITH_tests=ON -DWITH_examples=ON -DCMAKE_CXX_FLAGS_RELEASE=-DCGAL_NDEBUG -DCMAKE_PREFIX_PATH="C:\Miniconda\Library" $SOURCE_DIR  && ctest --build-config "Release" -L $(PACKAGE) -VV
```

Reply:
If you remove `-DCGAL_HEADER_ONLY=ON` then the `PATH` must contains the directory where CGAL DLLs are created.